### DISCUSSION TITLE
Only invoke async validator when other non-async rules are satisfied

### DISCUSSION THREAD

**Discussion by letmaik** [vueform/vueform#209] [Ideas]
https://vueform.com/docs/validating-elements#async-validation

```vue
<TextElement name="verificationCode" label="Verification Code" input-type="number" :rules="[
            'required',
            'size:4',
            verificationCodeCheck
          ]" />
```

`verificationCodeCheck` is a custom async validator that calls a backend API. I only want this to be invoked after the other validators have succeeded. Is this possible? Maybe a global setting that enforces that rules are evaluated in-order and only when previous rules succeeded would do the trick?

**Comment by adamberecz** [comment]
Sounds good!
